docs/03-03-runtime-operations.md

```diff
@@ -9,92 +9,37 @@ Kyma Environment Broker (KEB) allows you to configure operations that you can ru
-| BTPOperator_Cleanup          | BTP                             | Deletes service instances and service bindings from the cluster.                                                                                             | 
-| De-provision_AVS_Evaluations | AvS                             | Removes external and internal monitoring of Kyma Runtime.                                                                                                    |
-| EDP_Deregistration           | Event Data Platform             | Removes all Kyma runtime entries from the Event Data Platform.                                                                                                        |
-| IAS_Deregistration           | Identity Authentication Service | Removes the ServiceProvider from IAS.                                                                                                                        | 
-| Deregister_Cluster           | Reconciler                      | Removes the cluster from Reconciler.                                                                                                                     | 
-| Check_Cluster_Deregistration | Reconciler                      | Checks if the cluster deregistration is complete.                                                                                                            | 
-| Remove_Runtime               | Provisioner                     | Triggers deprovisioning of a Runtime in Runtime Provisioner.                                                                                             | 
-| Check_Runtime_Removal        | Provisioner                     | Checks if the cluster deprovisioning is complete.                                                                                                            |
-| Release_Subscription         | Subscriptions                   | Releases the subscription used by the cluster.<br/>                                                                                                               |
-| Remove_Instance              | Deprovisioning                  | Removes the instance from the database.                                                                                                                      |
->**NOTE:** The timeout for processing this operation is set to `24h`.
-
-## Upgrade Kyma
-
-Each upgrade step is responsible for a separate part of upgrading Kyma runtime dependencies. To properly upgrade SAP BTP, Kyma runtime, you need the data used during the provisioning. You can fetch this data from the **ProvisioningOperation** struct in the [initialization](https://github.com/kyma-project/kyma-environment-broker/blob/main/internal/process/upgrade_kyma/initialisation.go) step.
+There is only one step, which fails the operation - `Check_Runtime_Removal`. It fails the operation in case of a timeout when checking for the Provisioner to remove the shoot. 
```

Comment:
There is one step which is an exception. All others should not fail the operation in case of an error. That is a general suggestion or a rule. It is not strict rule, but writing a code which fails the deprovisioning operation should be done carefully. If someone writes a piece of code which fails deprovisioning operation - must know the consequences.